src/providers/TransactionBuilderContextProvider.tsx

```diff
@@ -644,7 +650,117 @@ const TransactionBuilderContextProvider = ({
       setTransactionBlocks([]);
       showAlertModal('Transaction sent');
       setIsSubmitting(false);
-    } else {
+
+    }
+    else if (crossChainActions[0].type == TRANSACTION_BLOCK_TYPE.PLR_DAO_STAKE) {
```

Comment:
```suggestion
    } else if (crossChainActions[0].type == TRANSACTION_BLOCK_TYPE.PLR_DAO_STAKE) {
```

Reply:
hmm this would mean that this would only work if FIRST action is `PLR_DAO_STAKE`, please check how other blocks are handled